app/views/edit/deployment-config.html

```diff
@@ -0,0 +1,373 @@
+                              </div>
+                            </div>
+                          </div>
+                          <div class="gutter-top">
+                            <div ng-if="!view.advancedStrategyOptions">To set additional parameters or edit lifecycle hooks, view <a href="" ng-click="view.advancedStrategyOptions = true">advanced strategy options</a></div>
+                            <a ng-if="view.advancedStrategyOptions" href="" ng-click="view.advancedStrategyOptions = false">Hide strategy advanced options</a>
+                          </div>
+                        </dl>
+                      </div>
+
+
+                      <div class="section">
+                        <h3 class="with-divider">Images</h3>
+                        <dl class="dl-horizontal left">
+
+                          <div ng-repeat="(containerName, containerData) in containerConfigByName">
+
+                            <div class="container-name">
+                              <h4>Container {{containerName}}</h4>
+                            </div>
+                            <div class="form-group" ng-if="containerData.hasDeploymentTrigger">
+                              <label class="required">Image Change Trigger</label>
+                              <istag-select model="containerData.triggerData.istag" select-disabled="disableInputs" include-shared-namespace="true"></istag-select>
+                              <span class="help-block">New deployments will automatically start when the image stream tag is updated.</span>
+                            </div>
+
+                            <div ng-if="!containerData.hasDeploymentTrigger">
+                              <label for="imageName" class="required">Image Name</label>
+                              <div>
+                                <input class="form-control"
+                                  id="imageName"
+                                  name="imageName"
+                                  ng-model="containerData.image"
+                                  type="text"
+                                  autocorrect="off"
+                                  autocapitalize="off"
+                                  spellcheck="false"
+                                  required>
+                              </div>
+                              <span class="help-block">New deployment can be triggered only manually.</span>
+                            </div>
+
+                            <div class="checkbox form-group">
+                              <label>
+                                <input type="checkbox" ng-model="containerData.hasDeploymentTrigger">
+                                Automatically start new deployments when the image stream tag is updated.
+                              </label>
+                            </div>
+
+                          </div>
+
+                          <div ng-if="view.advancedImageOptions">
```

Comment:
Since there's only one field we're hiding and showing, I wonder if it's even worth having the advanced link. (I know that's different than what we told you.)

@jwforres opinion?